dops/src/enum/user-role.enum.ts

```diff
@@ -0,0 +1,30 @@
+export enum ClientUserRole {
+  PERMIT_APPLICANT = 'PAPPLICANT',
+  COMPANY_ADMINISTRATOR = 'ORGADMIN',
+}
+
+export enum IDIRUserRole {
+  PPC_CLERK = 'PPCCLERK',
+  // PPC SUPERVISOR aka CTPO
+  CTPO = 'CTPO',
```

Comment:
Its PPC_SUPERVISOR in vehicles. I think we decided to go with PPC_SUPERVISOR

Reply:
Please confirm other enum keys are consistent among vehicles, dops. policy, and scheduler

Reply:
I've modified the enum key to be CTPO to make it consistent. Since CTPO is the primary way the supervisor is addressed by, we can keep it CTPO for now.